plugins/commands/compare/src/main/kotlin/CompareCommand.kt

```diff
@@ -92,7 +96,14 @@ class CompareCommand : OrtCommand(
             throw ProgramResult(2)
         }
 
-        val deserializer = fileA.mapper()
+        val deserializer = fileA.mapper().registerModule(
```

Comment:
Note that making the results invariant already on deserialization should also work for the existing `TEXT_DIFF` method, which currently makes results only invariant on serialization.

That is, with this code, I believe the existing code

                val serializer = deserializer.copy().registerModule(
                    SimpleModule().apply {
                        if (ignoreTime) addSerializer(InvariantInstantSerializer())
                        if (ignoreEnvironment) addSerializer(InvariantEnvironmentSerializer())
                    }
                )

and the referenced serializers should not be required anymore.

Phrased differently, we should make results invariant either on deserialization or on serialization, but not both.

Reply:
I would drop the serializers, because making it invariant on deserialization works for both methods.